CHANGELOG.md

```diff
@@ -7,7 +7,27 @@ Then it was renamed Kiwix-html5 (and uses ZIM files), then was renamed Kiwix-JS.
 ## Kiwix-JS v2.2.0
 Released on TODO
 
-TODO
+Rename the project to Kiwix-JS instead of Kiwix-html5.
+
+Compatibility with splitted English wikipedia ZIM files (which now has more than 26 files).
```

Comment:
Change "splitted" to "split" (irregular past participle!), and change "which now has" to "which now have"

Reply:
OK thanks : I'll fix that everywhere I had put this word "splitted"